app/controllers/api/v1/road_trip_controller.rb

```diff
@@ -1,8 +1,13 @@
 class Api::V1::RoadTripController < ApplicationController
 
   def create
+    require 'pry'; binding.pry
```

Comment:
Probably could abstract this more but then when would the abstraction ever stop??